### PR TITLE
lookup: re-add fastify

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

**Comment by Trott** [comment]
> @Trott how can I test this in CITGM smoker?

If I didn't make any mistakes, it's running here: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3227/

Here are the steps I took to do it:

1. Go to https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/.
2. Select "Build With Parameters" on the left.
3. Enter `refs/pull/973/head` for the GIT_REMOTE_REF
4. Enter `citgm fastify` for the CITGM_COMMAND (so that it only runs a test for fastify and not every module in lookup.json)
5. Use the Build button

**Comment by Trott** [comment]
> > @Trott how can I test this in CITGM smoker?
> 
> EDIT: It looks like the below stuff didn't work so I guess I didn't quite get it right. The GIT_REMOTE_REF probably refers to the nodejs/node repository and not the nodejs/citgm repository....

Maybe someone on @nodejs/citgm knows the right thing to do to test this PR?

**Comment by targos** [comment]
https://github.com/nodejs/citgm/actions/runs/6148435347

I used https://github.com/nodejs/citgm/actions/workflows/test-module.yml

CleanShot 2023-09-11 at 17 21 38@2x

File: lib/lookup.json [hunk 1]
```diff
@@ -165,6 +165,8 @@
   "fastify": {
     "maintainers": ["mcollina", "delvedor"],
     "prefix": "v"
```
Comment (targos) [inline lib/lookup.json:167]:
```suggestion
    "prefix": "v",
```

**Comment by richardlau** [comment]
> > > @Trott how can I test this in CITGM smoker?
> > 
> > 
> > EDIT: It looks like the below stuff didn't work so I guess I didn't quite get it right. The GIT_REMOTE_REF probably refers to the nodejs/node repository and not the nodejs/citgm repository....
> 
> Maybe someone on @nodejs/citgm knows the right thing to do to test this PR?

If you want to use the Jenkins job, use the **CITGM** parameter:
image
e.g. for this PR, try `nodejs/citgm#fastify-fix`.

**Comment by targos** [comment]
New run passed: https://github.com/nodejs/citgm/actions/runs/6148435347

**Comment by mcollina** [comment]
Let's see what the smoker says: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3230/.

**Review by ljharb — APPROVED** [review]

**Comment by mcollina** [comment]
Fresh CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3255/

**Comment by mcollina** [comment]
Fresh CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3272/

File: lib/lookup.json [hunk 2]
```diff
@@ -153,6 +153,12 @@
     "prefix": "v",
     "maintainers": "dougwilson"
   },
+  "fastify": {
+    "maintainers": ["mcollina"],
+    "prefix": "v",
+    "skip": ["s390x", "aix", "aix72-ppc64"],
```
Comment (richardlau) [inline lib/lookup.json:159]:
```suggestion
    "skip": ["s390x", "aix"],
```

You don't need `aix72-ppc64` if `aix` is also in this set. Skips are only considered for `citgm-all` -- if you're running `citgm <module>` CITGM assumes you wanted to run for that module.

Comment (mcollina) [reply lib/lookup.json:159]: Ah awesome, so I don't need a full green run.

**Comment by simone-sanfratello** [comment]
Hello, I'm investigating the failing tests, tackling one by one.

For example this one

https://github.com/fastify/fastify/blob/main/test/custom-http-server.test.js#L11

Looks like it gets timeout on dns lookup

```js
const localAddresses = await dns.lookup('localhost', { all: true })
```

I'm adding more logs to get information, btw, what the dns lookup is suppose to return?

**Comment by mcollina** [comment]
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3313/

**Comment by mcollina** [comment]
CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3326/

**Comment by mcollina** [comment]
CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3335/

**Comment by mcollina** [comment]
@targos can you confirm everything is passing correctly? I think we got there eventually.

**Comment by targos** [comment]
LGTM

**Pull request merged**